.gitignore

```diff
@@ -1,7 +1,10 @@
-*~
-playbook.retry
+__pycache__
+.coverage
+.mypy_cache
+.pytest_cache
 .python-version
 .terraform
-terraform.tfstate*
 *.tfvars
-*.dyn.tf
```

Comment:
Do we still need the `*.dyn.tf` line for the code generated in the `terraform/dyn_*` directories?  I'm not sure, so I'm asking.

Reply:
Ah, that's a good question. We probably do and I just got overzealous with `.gitignore` cleaning.

Reply:
I have restored that pattern in 22c028b